packages/block-editor/src/components/list-view/drop-indicator.js

```diff
@@ -68,40 +68,41 @@ export default function ListViewDropIndicator( {
 		};
 	}, [ getDropIndicatorIndent, targetElement ] );
 
-	const getAnchorRect = useCallback( () => {
-		if ( ! targetElement ) {
-			return {};
+	const popoverAnchor = useMemo( () => {
+		const isValidDropPosition =
+			dropPosition === 'top' ||
+			dropPosition === 'bottom' ||
+			dropPosition === 'inside';
```

Comment:
Change no. 2: instead of returning an empty object at the end of this function if `dropPosition` didn't have the required values, I moved that check at the start of the function.